shanoir-ng-front/src/app/subjects/subject/subject.component.ts

```diff
@@ -222,9 +222,9 @@ export class SubjectComponent extends EntityComponent<Subject> implements OnInit
 
     public toggleAnonymised() {
         if (this.isAlreadyAnonymized && this.subjectNamePrefix) {
-            this.subject.name = this.subjectNamePrefix;
-        } else if (!this.isAlreadyAnonymized && this.subjectNamePrefix && this.dicomPatientName) {
             this.subject.name = this.subjectNamePrefix + this.dicomPatientName + '-';
```

Comment:
This adds a "-" at the end of the subject name:
![image](https://user-images.githubusercontent.com/7604176/149168398-26f1f3e8-0cd2-4221-9616-293f4174b1b3.png)

Is it on purpose ? If it's already anonymised, ha may want to let the name as is.